extensions/big-data-cluster/src/wizards/create-cluster/resourceStrings.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as nls from 'vscode-nls';
+
+const localize = nls.loadMessageBundle();
+
+export const WizardTitle = localize('bdc-create.wizardTitle', 'Create a 2019 Big Data cluster');
+export const GenerateScriptsButtonText = localize('bdc-create.generateScriptsButtonText', 'Generate Scripts');
+export const CreateClusterButtonText = localize('bdc-create.createClusterButtonText', 'Create');
+
+//#region cluster profile page strings
+
+export const ClusterProfilePageTitle: string = localize('bdc-create.clusterProfilePageTitle', 'Select a cluster profile');
+export const ClusterProfilePageDescription: string = localize('bdc-create.clusterProfilePageDescription', 'Select your requirement and we will provide you a pre-defined default scaling. You can later go to cluster configuration and customize it.');
+
+//#endregion
+
+//#region settings page strings
+
+export const SettingsPageTitle: string = localize('bdc-create.settingsPageTitle', 'Settings');
+export const SettingsPageDescription: string = localize('bdc-create.settingsPageDescription', 'Configure the settings required for deploying SQL Server big data cluster');
+export const BasicSettingsText: string = localize('bdc-create.BasicSettingsText', 'Basic Settings');
+export const AdminUserNameText: string = localize('bdc-create.AdminUserNameText', 'Admin user name');
```

Comment:
Do we capitalize Username / show it as 1 word elsewhere? most other settings are capitalized

Reply:
it seems username is indeed a word now, I'll fix it.